app/src/main/res/values/strings.xml

```diff
@@ -572,4 +572,17 @@
   <string name="administrator_controls_fragment_test_activity_label">Administrator Controls Fragment Test Activity</string>
   <string name="select_a_topic_to_start">Select a Topic to Start</string>
   <string name="revision_navigation_cards_header">Continue Studying</string>
+  <string name="info_tab_content_description">Info tab</string>
+  <string name="revision_tab_content_description">After completing a lesson, revise your concepts in the revision tab.</string>
```

Comment:
Is this for the spotlight? If so, then perhaps "Revise important concepts from the lessons." since this shows all such concepts, not just concepts corresponding to completed lessons.